Editor/SettingsAnalysis/PlayerSettingsAnalyzer.cs

```diff
@@ -79,13 +81,28 @@ class PlayerSettingsAnalyzer : ISettingsModuleAnalyzer
             messageFormat = "Player: C++ Compiler Configuration is set to 'Debug'"
         };
 
+        private static readonly Descriptor k_LightmapStreamingEnabledDescriptor = new Descriptor(
+            PAS1006,
+            "Player: Lightmaps Streaming Disabled",
+            new[] { Area.GPU, Area.CPU },
+            "<b>Lightmap Streaming</b> is not enabled. This may lead to a increase of the GPU memory",
```

Comment:
Sorry, just English grammar:  The second sentence could be:

"This may lead to an increase of GPU memory"

("an" instead of "a"; GPU memory doesn't need article "the")